packages/recurring-payments-job/src/__tests__/recurring-payments-processor.spec.js

```diff
@@ -70,9 +70,26 @@ const getMockDueRecurringPayment = ({ agreementId = 'test-agreement-id', id = 'a
   expanded: { activePermission: { entity: { referenceNumber } } }
 })
 
+const makeSendPaymentResponse = (overrides = {}) => ({
+  payment_id: 'pay-1',
+  created_date: '2025-01-01T00:00:00.000Z',
+  ...overrides
+})
+
 describe('recurring-payments-processor', () => {
   const [{ value: debugLogger }] = db.mock.results
 
+  const date1 = '2025-01-01T00:00:00.000Z'
+  const date2 = '2025-01-01T00:01:00.000Z'
+
+  const mockTwoDuePayments = () => {
```

Comment:
Rather than setting this up as a separate function, I would suggest putting it in as setup for a `before` block. So you could nest another `describe('when multiple payments are due')` block within `describe('recurring-payments-processor')`, move your tests that need this mocking into that, and then do a `beforeEach` to set up the mocking. I think that'd be more consistent with how we've done our setup for tests elsewhere in the service.